### PR TITLE
Bug in comments where rating was over ridden by description

### PR DESCRIPTION
**Pull request by nkeller1**
### Type of change made:
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling/UX

### Detailed Description
When a get request was being returned there was an issue where the rating was being overridden by the description. Bug traced and corrected.

Alongside this bug change am pushing up campsite1 and campsite2 to include comments in the seed file.

### Why is this change required? What problem does it solve?
Bug. Broken Code.

### Were there any challenges that arose while implementing this feature? If so, how were they resolved?
N/a

### Test coverage snapshot:
N/A

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -168,7 +168,7 @@ def comments(id, **kwargs):
 					'id': comment.id,
 					'title': comment.title,
 					'description': comment.description,
-					'rating': comment.description
+					'rating': comment.rating
```
Comment (nkeller1) [inline app/__init__.py:171]: Bug was here.

File: manage.py [hunk 2]
```diff
@@ -34,6 +34,10 @@ def seed():
     campsite1.amenities.append(fire)
     campsite1.amenities.append(hike)
     campsite1.amenities.append(bike)
+    comment1 = Comment(title = 'Best Biking in Western Colorado', description = 'It gets hot during the day but the campsites are right off the trail so you can go back to camp and cool off.', rating = "5")
+    comment2 = Comment(title = 'Scorpians!', description = 'I did not know scorpians swarmed. At our camp they do...', rating = "3")
+    campsite1.comments.append(comment1)
```
Comment (nkeller1) [inline manage.py:39]: Add comments to campsite1

File: manage.py [hunk 3]
```diff
@@ -47,6 +51,10 @@ def seed():
         )
     campsite2.amenities.append(fire)
     campsite2.amenities.append(hike)
+    comment3 = Comment(title = 'The Views!', description = 'The dispersed camping has eveything you need.', rating = "5")
+    comment4 = Comment(title = 'The camping areas flood.', description = 'Do not go if there is rain in forecast.', rating = "5")
+    campsite2.comments.append(comment3)
```
Comment (nkeller1) [inline manage.py:56]: add comments to campsite2

File: app/__init__.py (hunk 1 above)
Comment (hborrelli1) [reply app/__init__.py:171]: Sweet, thanks for fixing this!

**Pull request merged**